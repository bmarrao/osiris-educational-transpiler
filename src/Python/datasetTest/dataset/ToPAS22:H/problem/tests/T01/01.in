4
4 3
Amadeu Beatriz Carlos Daniel
Amadeu Carlos
Beatriz Carlos
Daniel Beatriz
4 4
Amadeu Beatriz Carlos Daniel
Amadeu Beatriz
Amadeu Carlos
Beatriz Carlos
Daniel Beatriz
4 0
Rita Tiago Manuel Susana
2 1
Paulo Raquel
Paulo Raquel
