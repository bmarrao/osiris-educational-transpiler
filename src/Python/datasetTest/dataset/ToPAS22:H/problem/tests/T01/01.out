Possivel: 8
Impossivel
Possivel: 24
Impossivel
